themes/powerline/powerline.theme.sh

```diff
@@ -7,6 +7,7 @@ POWERLINE_LEFT_SEPARATOR=${POWERLINE_LEFT_SEPARATOR:=""}
 
 USER_INFO_SSH_CHAR=${POWERLINE_USER_INFO_SSH_CHAR:=" "}
 USER_INFO_THEME_PROMPT_COLOR=32
+USER_INFO_THEME_PROMPT_TEXT_COLOR="#4c566a"
```

Comment:
- When you add a new configuration variable of the powerline theme, could you also define it in the other powerline themes (`powerline-*`)?
- Also, could you make the default value within the 256 index colors? Not all the terminals support 24-bit color specifications for SGR. The terminals without the support for 24-bit color will misinterpret the R/G/B values as independent SGR codes, whose results could be disastrous. The 24-bit colors should only be enabled when the user explicitly requests them. More specifically, you can override the variable value after sourcing `oh-my-bash.sh` in `~/.bashrc`.